lib/aruba/api.rb

```diff
@@ -98,12 +98,54 @@ def directory?(file)
       File.directory? expand_path(file)
     end
 
+    # Check if path is absolute
+    #
+    # @return [TrueClass, FalseClass]
+    #   Result of check
+    def absolute?(path)
+      Pathname.new(path).absolute?
+    end
+
+    # Check if path is relative
+    #
+    # @return [TrueClass, FalseClass]
+    #   Result of check
+    def relative?(path)
+      Pathname.new(path).relative?
+    end
+
     # Return all existing paths (directories, files) in current dir
     #
     # @return [Array]
     #   List of files and directories
     def all_paths
```

Comment:
Why do we need this method? If we have `list` and `expand_path` I think people can compose this themselves. Seems like API noise to me.